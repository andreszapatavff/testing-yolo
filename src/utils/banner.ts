import figlet from 'figlet';

export const YOLOBANNER = (input: string) => {
  return figlet.textSync(`YOLO: \n${input}`, {
    horizontalLayout: 'default',
    verticalLayout: 'default',
    width: 80,
    whitespaceBreak: true,
  });
};
