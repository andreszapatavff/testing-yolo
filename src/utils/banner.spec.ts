import { YOLOBANNER } from './banner';
import { test, expect } from 'vitest';

test('YOLOBANNER', () => {
  expect(YOLOBANNER('Hello World!')).toMatchSnapshot();
});
