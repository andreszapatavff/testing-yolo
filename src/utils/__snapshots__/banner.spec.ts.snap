// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`YOLOBANNER 1`] = `
" __   _____  _     ___                                 
 \\ \\ / / _ \\| |   / _ \\ _                              
  \\ V / | | | |  | | | (_)                             
   | || |_| | |__| |_| |_                              
  _|_|_\\___/|_____\\___/(_)        __         _     _ _ 
 | | | | ___| | | ___   \\ \\      / /__  _ __| | __| | |
 | |_| |/ _ \\ | |/ _ \\   \\ \\ /\\ / / _ \\| '__| |/ _\` | |
 |  _  |  __/ | | (_) |   \\ V  V / (_) | |  | | (_| |_|
 |_| |_|\\___|_|_|\\___/     \\_/\\_/ \\___/|_|  |_|\\__,_(_)
                                                       "
`;
